apps/code-examples/src/app/code-examples/ag-grid/data-grid/data-manager/data-manager-data-grid-docs-demo-view-grid.component.ts

```diff
@@ -72,80 +70,78 @@ export class DataManagerDataGridsDocsDemoViewGridComponent implements OnInit {
     },
   ];
 
+  private columnPickerOptions: SkyDataManagerColumnPickerOption[] = [
+    {
+      id: 'name',
+      label: 'Name',
+      description: 'The name of the employee.',
+    },
+    {
+      id: 'age',
+      label: 'Age',
+      description: 'The age of the employee.',
+    },
+    {
+      id: 'startDate',
+      label: 'Start date',
+      description: 'The start date of the employee.',
+    },
+    {
+      id: 'endDate',
+      label: 'End date',
+      description: 'The end date of the employee.',
+    },
+    {
+      id: 'department',
+      label: 'Department',
+      description: 'The department of the employee',
+    },
+    {
+      id: 'jobTitle',
+      label: 'Title',
+      description: 'The job title of the employee.',
+    },
+  ];
+
   public dataState = new SkyDataManagerState({});
 
   public columnApi?: ColumnApi;
   public displayedItems: any[] = [];
   public gridApi?: GridApi;
-  public gridInitialized = false;
+  public isGridReadyForInitialization = false;
   public gridOptions!: GridOptions;
   public noRowsTemplate: string;
-  public isActive = false;
+  public isThisViewActive = false;
 
   public viewConfig: SkyDataViewConfig = {
     id: this.viewId,
     name: 'Data Grid View',
     icon: 'table',
     searchEnabled: true,
     sortEnabled: true,
-    multiselectToolbarEnabled: true,
+    multiselectToolbarEnabled: false,
```

Comment:
I believe you can leave this out and its disabled by default

Reply:
As our team standard, do we prefer implicit configuration over explicit?

Reply:
if so, can we put that in the team wiki somewhere?

Reply:
regardless, it's done in: [remove explicit configuration](https://github.com/blackbaud/skyux/pull/531/commits/d302ff5da8384ffb078d5d8b86361bce1aa996f1)